modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -4730,6 +4730,20 @@ CV_EXPORTS_W Size getTextSize(const String& text, int fontFace,
                             double fontScale, int thickness,
                             CV_OUT int* baseLine);
 
+
+/** @brief Calculates the font-specific size to use to achieve a given height in pixels.
```

Comment:
Could you please add usage sample as we have for ```getTextSize```

Reply:
Thanks for reviewing. Yes, I can. Should I push to this branch or open a new one as it is already merged?

Reply:
Yes, I think we need this in separate pull request.